application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -84,9 +91,11 @@
 
     private static final List<OptionData> inviteOptions = List.of(
             new OptionData(OptionType.STRING, MAX_USES_OPTION,
-                    "The amount of times the invite can be used, default is infinity", false),
+                    "The amount of times the invite can be used, default is infinity", false)
+                        .setRequiredRange(MIN_USE_AMOUNT, MAX_USE_AMOUNT),
             new OptionData(OptionType.INTEGER, MAX_AGE_OPTION,
-                    "Max age in seconds. Set this to 0 to never expire, default is 1 day", false));
+                    "Max age in seconds. Set this to 0 to never expire, default is 1 day", false)
+                        .setRequiredRange(MIN_AGE_DURATION, MAX_AGE_DURATION));
```

Comment:
Since you are using min and max now, is the `handleIntegerTypeOption` still needed? If not, remove. (@Tais993 please confirm, thanks)

Reply:
The logic checking or it's a valid Integer isn't needed, but the null checks and such still are.

Reply:
Okay, then lets just adjust it a bit and all cool, I guess.